README.md

```diff
@@ -1,3 +1,16 @@
 # otelhttp
 
-This is a copy from [opentelemetry-go-contrip](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation)
+This is a copy from [opentelemetry-go-contrib](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation)
+
+The README there says, in part
+
+> The last place instrumentation should be hosted is here in this repository. Maintaining instrumentation here hampers the development of OpenTelemetry for Go and therefore should be avoided.
+
+There have been multiple PRs to improve the net/http instrumentation
+that have not been merged for months and so, in typical go fashion, we
+have to re-implement it ourselves.
+
+[otelhttp](https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/net/http/otelhttp)
+
+as of [v.0.26.1](https://github.com/open-telemetry/opentelemetry-go-contrib/commit/7876cd14dc5f09765205caa0fb420fafe23141aa)
+
```

Comment:
Can you add setup instructions? Especially for non-nix users since they'll have to make sure `go` and `pre-commit` are set up correctly. Well even `nix` users will need to install `pre-commit` hooks.

Reply:
I added a blurb to the README